packages/appstore/src/components/wallet-account/wallet-account.scss

```diff
@@ -0,0 +1,108 @@
+.wallet-account {
+    position: relative;
+    background: #fff8f9;
+
+    &__background {
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        height: 15.8rem;
+    }
+
+    &__header {
+        height: 100%;
+        position: relative;
+        border-radius: 1.6rem;
+        overflow: hidden;
+        padding: 2.4rem;
+        display: flex;
+        align-items: center;
+    }
+
+    &__apps-wrapper {
+        display: flex;
+        flex-direction: column;
+    }
+
+    &__apps-title {
+        display: flex;
+        align-items: center;
+        margin-top: 2rem;
+        margin-bottom: -1rem;
+
+        &--link {
+            margin-left: 1.5rem;
+        }
+
+        &.no-mt {
+            margin-top: 0 !important;
+        }
+    }
+
+    &__apps {
+        display: flex;
+        flex-wrap: wrap;
+        gap: 3rem;
+    }
+
+    &__app-launcher-button {
+        background-color: #ff444f29 !important;
```

Comment:
Can we use hex color code for this and add this to shared as well?

Reply:
done :heavy_check_mark: